### PR TITLE
Documentation update for options missing in targetGlobalSettings()

### PR DESCRIPTION
**Pull request by ericfichtel**
## Description

The solution to the customer issue logged in the ticket below is to configure the `pollingInterval` setting in `targetGlobalSettings()`, however that was missing from the documentation. This PR adds that setting, along with `propertyToken` and `artifactLocation` to the documentation. 

## Related Issue

https://jira.corp.adobe.com/browse/TNT-46359

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by ericfichtel** [comment]
Closing to add reviews to a new pull request

File: src/pages/implement/client-side/atjs/atjs-functions/targetglobalsettings.md [hunk 1]
```diff
@@ -212,6 +218,18 @@ The deviceIdLifetime setting is overrideable in at.js version 2.3.1 or later.
 * **Default Value**: true
 * **Description**: When enabled, automatically retrieve experiences that must be returned on page load.
 
+### pollingInterval
+
+* **Type**: Number
+* **Default Value**: 300000 (Five minutes in milliseconds)
+* **Description**: Interval that at.js fetches a new version of an artifact and updates the cache
+
+### propertyToken
```
Comment (dcottingham) [inline src/pages/implement/client-side/atjs/atjs-functions/targetglobalsettings.md:227]: I don't think we'll need to add propertyToken to targetGlobalSettings after all.  I looked into [the code](https://git.corp.adobe.com/TnT/atjs-library-v2/blob/2d92da412c01ddffceabb5236feab12956d04370/src/delivery/params/index.js#L83) and it looks like if user wants to set a property in at.js they would use the `targetPageParams.at_property`

File: src/pages/implement/client-side/atjs/atjs-functions/targetglobalsettings.md [hunk 2]
```diff
@@ -212,6 +218,18 @@ The deviceIdLifetime setting is overrideable in at.js version 2.3.1 or later.
 * **Default Value**: true
 * **Description**: When enabled, automatically retrieve experiences that must be returned on page load.
 
+### pollingInterval
+
+* **Type**: Number
+* **Default Value**: 300000 (Five minutes in milliseconds)
+* **Description**: Interval that at.js fetches a new version of an artifact and updates the cache
```
[the remark this answers is not in the document]
Comment (dcottingham) [reply src/pages/implement/client-side/atjs/atjs-functions/targetglobalsettings.md:225]: Let's also specify for users that 300000 is also the minimum value - you can verify that [here](https://github.com/adobe/target-nodejs-sdk/blob/2635d7dfb06554f5bd282ae399140d3c21723388/packages/target-decisioning-engine/src/constants.js#L8).

**Review by dcottingham — CHANGES_REQUESTED** [review]

File: src/pages/implement/client-side/atjs/atjs-functions/targetglobalsettings.md [hunk 3]
```diff
@@ -176,15 +182,15 @@ The deviceIdLifetime setting is overrideable in at.js version 2.3.1 or later.
 
 ### imsOrgId
 
-* **Type**: Sting 
+* **Type**: String
```
[the remark this answers is not in the document]
Comment (jasonwaters) [reply src/pages/implement/client-side/atjs/atjs-functions/targetglobalsettings.md:185]: sting-wink-gif

[the remark this answers is not in the document]
Comment (ericfichtel) [reply src/pages/implement/client-side/atjs/atjs-functions/targetglobalsettings.md:185]: I don't drink coffee I take tea

**Review by dcottingham — APPROVED** [review]
Changes look good to me

**Comment by ericfichtel** [comment]
Hi @judy-kim looks like we are good from our side - would you be able to review this when you have the chance? Thanks!

**Review by judy-kim — APPROVED** [review]

**Pull request merged**